src/main/java/hu/oe/nik/szfmv/automatedcar/virtualfunctionbus/packets/HMIOutputPackets/BaseInputPacket.java

```diff
@@ -0,0 +1,19 @@
+package hu.oe.nik.szfmv.automatedcar.virtualfunctionbus.packets.HMIOutputPackets;
+
+import hu.oe.nik.szfmv.automatedcar.systemcomponents.Shitfer;
+
+public interface BaseInputPacket {
+
+    int getGasPedalValue();
+    int getBreakPedalValue();
+    int getSteeringWheelValue();
+    int getIndexValue(/*negatív -> balra, pozitív -> jobbra, 0 -> kikapcsolva */);
```

Comment:
Associating numbers with states are really C'ish. Should use an `enum` instead in these cases.